ground/gcs/src/plugins/uavobjectwidgetutils/configtaskwidget.cpp

```diff
@@ -415,24 +419,27 @@ void ConfigTaskWidget::addApplySaveButtons(QPushButton *update, QPushButton *sav
         connect(smartsave, SIGNAL(saveSuccessfull()), this, SLOT(clearDirty()));
         connect(smartsave, SIGNAL(beginOp()), this, SLOT(disableObjUpdates()));
         connect(smartsave, SIGNAL(endOp()), this, SLOT(enableObjUpdates()));
+
+        foreach (objectToWidget *oTw, objOfInterest) {
```

Comment:
Note: Q_FOREACH will die at some point, since it's superseded by stdlib since c++11 (ranged for).

Reply:
Yah, this is just moved up (I forget why I moved it up) and had a test added inside.

Reply:
Ack. Just noting.